sky/engine/core/dart/painting.dart

```diff
@@ -726,34 +756,9 @@ class Canvas extends NativeFieldWrapperClass2 {
     if (colors.isNotEmpty && colors.length != vertexCount)
```

Comment:
similarly here. we check that colors isn't empty and has the right count, but we don't check it's not null.